nuitka/distutils/Build.py

```diff
@@ -23,6 +23,20 @@ class NuitkaBuildMetaBackend(setuptools.build_meta._BuildMetaBackend):
     def build_wheel(
         self, wheel_directory, config_settings=None, metadata_directory=None
     ):
+        # Allow falling back to setuptools when the `no-nuitka` configuration setting is set to true.
+        if config_settings:
+            no_nuitka = config_settings.pop("no-nuitka", "false").lower()
```

Comment:
We got for `setup.cfg`, and `setup` already the naming 

```
   [metadata]
   build_with_nuitka = true
```

So I guess, I would like to stick that name. Ordinarily, you could just switch the backend back to disable it with the pyproject, but I can see how that's less obvious.

Reply:
So you propose to invert the logic and use `build_with_nuitka=false` as the config setting? I'm fine either way. Also, feel free to edit the MR as you see fit. I would really love to see this merged, since it would simplify our CI pipeline :-)

Reply:
Yes, effectively that.